docs/streams/upgrade-guide.html

```diff
@@ -191,6 +191,17 @@ <h3><a id="streams_api_changes_210" href="#streams_api_changes_210">Streams API
         details, see <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-312%3A+Add+Overloaded+StreamsBuilder+Build+Method+to+Accept+java.util.Properties">KIP-312</a>
     </p>
 
+    <p>
+        We are introducing static membership towards KStream users. This feature reduces unnecessary rebalances during normal application upgrades or rolling bounces.
+        If you want to use static membership, first thing is to upgrade both broker cluster and stream apps to 2.3.0. This involves a necessity to change <code>inter.broker.protocol.version</code>
```

Comment:
`2.3.0` -> `2.3 or beyond`: we do not change web docs when release new bug fix versions e.g. 2.3.1 so we usually avoid indicating the specific bug-fix digit unless necessary.

Reply:
`This involves a necessity to change ...` -> `to 2.3 or beyond, and also make sure the upgraded brokers are using <code>inter.broker.protocol.version</code> of 2.3 or beyond as well.`